handbook/business-operations/README.md

```diff
@@ -904,6 +904,7 @@ These groups maintain the following [Slack channels](https://fleetdm.com/handboo
 | `#help-onboarding`                      | Mike McNeil
 | `#help-manage`                          | Mike McNeil
 | `#help-open-core-ventures` _(¶¶)_       | Mike McNeil
+| `#g-security`                           | Head of security
```

Comment:
Since security isn't a department or product group, we can leave off the `g-` prefix for this channel.  Since the channel is a feed, we can use the `__` prefix convention for it.  One-off questions about security can go to #g-business-operations.

Will DRI be Joanne or Zach?

Reply:
This would require editing our security training video (which references the channel by that name). Does that seem worth it?

Reply:
Good point!  We can take the damage.  It's OK for the training video to have the wrong channel name (it's inevitable it will get out of date as handbook and processes change).  In future videos, we can demo a search in the handbook instead.

For DRI, let's identify the name of the individual, to disambiguate acting head of security

Reply:
@Sampfluger88 g-security channel says:

> Please report any security incidents or concerns here. For suspected phishing emails, please use the "report phishing" button in Google. If you think you may have clicked on a malicious link in an email, please report that here.

…as its description.  So instead, that responsibility will fall to the #g-business operations channel, and:
- [ ] we'll rename #g-security to `_security`
- [ ] we'll update the `#_security` channel description and/or topic in Slack to be about being a feed instead of what it is today
- [ ] we'll update this PR to match
- [ ] we'll update this PR to make Zach Wasserman DRI of `_security` (and `#g-business-operations` is already covered with Joanne as DRI in the bizops handbook)

Reply:
<img width="1458" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/f0ab2378-02dd-4f4b-9a46-d995ad5839c1">
<img width="540" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/bc9c1472-4b6b-4565-bc84-21ff2dcb5ba4">